Add external link to starlette documentation for further details (#1717)
fastapi/fastapi: docs/en/docs/advanced/testing-websockets.md

```Python hl_lines="27 28 29 30 31"
{!../../../docs_src/app_testing/tutorial002.py!}
```

> ## More info

Reviewer: ```suggestion:-0+1
## More info
```

> ## More info

Reviewer: ```suggestion
```

> For further deatails, check Starlette's documentation for <a href="https://www.starlette.io/testclient/#testing-websocket-sessions" class="external-link" target="_blank">testing WebSockets</a>.

Reviewer: ```suggestion
!!! note For further details, check Starlette's documentation for <a href="https://www.starlette.io/testclient/#testing-websocket-sessions" class="external-link" target="_blank">testing WebSockets</a>.
```
